packages/databases-collections/.depcheckrc

```diff
@@ -8,3 +8,5 @@ ignores:
  - "@types/chai-dom"
  - "@types/react"
  - "@types/react-dom"
+ignore-patterns:
+  - 'dist'
```

Comment:
why does this matter? if it does should we create a ticket to fix it everywhere?

Reply:
I will be removing this change from this PR and instead follow up with what we discussed regarding webpack plugin.